src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerMainTest.java

```diff
@@ -134,4 +134,24 @@ void givenCaseCannotBeListedRightAwayAndDirectionsAreBeingMade_thenNoMessagesSen
         assertThat(response.getErrors()).isEmpty();
     }
 
+    @DisplayName("When we have written an adjournment notice and excluded some panel members, add them "
+        + "to the excluded panel members list")
+    @Test
+    void givenPanelMembersExcluded_thenAddPanelMembersToExclusionList() {
+        panelMembersExcludedWithExistingExclusions();
+
+        assertThat(sscsCaseData.getSchedulingAndListingFields()
+            .getPanelMemberExclusions().getExcludedPanelMembers()).hasSize(3);
+    }
+
+    @DisplayName("When we have written an adjournment notice and excluded some panel members, and there are no current "
+        + "exclusions,  add them to the excluded panel members list")
+    @Test
+    void givenNoExistingPanelMembersExcluded_thenAddPanelMembersToExclusionList() {
+        panelMembersExcluded();

```

Comment:
Please inline the code in this test.
```suggestion
        sscsCaseData.getAdjournment().setPanelMembersExcluded(AdjournCasePanelMembersExcluded.YES);
        sscsCaseData.getAdjournment().setPanelMember1(JudicialUserBase.builder().idamId("1").build());
        sscsCaseData.getAdjournment().setPanelMember3(JudicialUserBase.builder().idamId("3").build());
        sscsCaseData.getAdjournment().setCanCaseBeListedRightAway(NO);
        sscsCaseData.getAdjournment().setAreDirectionsBeingMadeToParties(NO);

        handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);

```

Reply:
done